### PR TITLE
"Meine Akte"-Ansicht 

### PR DESCRIPTION
**Pull request by PotSchmoell**

### CODE REVIEW

File: src/app/my-file/components/my-file/my-file.component.scss [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    padding-top: 10%;
+    padding-bottom: 20%;
+    width: 100%;
+    height:100%;
+  margin: auto;
+}
+
+.app-file-container h2{
+    font-size: 100% !important;
+}
```
Comment (SandroGlueck) [inline src/app/my-file/components/my-file/my-file.component.scss:11]: What is this used for?

Comment (PotSchmoell) [reply src/app/my-file/components/my-file/my-file.component.scss]: Nothing. Leftover from trying to adapt Heading Size in file component.

File: src/app/core/components/header/header.component.html [hunk 2]
```diff
@@ -13,27 +13,19 @@
...
+          <button class="button-primary">
+            <div class="button-wrapper" [routerLink]="['/archive']">
+              <p>
+                Archiv
+              </p>
+              <mat-icon aria-hidden="false" aria-label="Example home icon">
+                archive
+              </mat-icon>
+            </div>
+          </button>
```
Comment (SandroGlueck) [inline src/app/core/components/header/header.component.html:26]: Might be an option to create a separate component for that.
Also rather sth for the future

**Review by SandroGlueck — APPROVED** [review]
Just some ideas for the future.

File: src/app/core/components/header/header.component.html [hunk 3]
```diff
@@ -13,27 +13,19 @@
...
         </li>
-        <div *ngIf="userLoaded" class="user-icon" [matMenuTriggerFor]="menu">
-          {{user?.name?.charAt(0)}}
+        <div *ngIf="userLoaded">
+          <button class="button-primary">
+            <div class="button-wrapper" [routerLink]="['/archive']">
+              <p>
+                Archiv
+              </p>
+              <mat-icon aria-hidden="false" aria-label="Example home icon">
```
[the remark this answers is not in the document]
Comment (PotSchmoell) [reply src/app/core/components/header/header.component.html]: Thanks for the hint. I removed them.

File: src/app/core/components/header/header.component.html (hunk 2 above)
Comment (PotSchmoell) [reply src/app/core/components/header/header.component.html]: Definitely!

**Pull request merged**